### ISSUE TITLE
Object Mapper Unification

### ISSUE THREAD

**Issue by krislefeber** [Majlanky/couch-slacker#71]
I want to start by saying I'm really impressed with this library.

One use case I've discovered to be missing is that the ObjectMapper class that is passed to the client is not used by all serialization/deserialization related functions. The reason for this, is that I'd like to be able to use Java 8 time classes(Instant, ZonedDateTime, etc.) but I run into deserialization issues and am not able to override the ObjectMapper that gets newed up in the function.

I'd be willing to look into the complexity of unifying all of the instances where ObjectMapper is used and allowing a user of the library to specify their own for all serialization work. Before starting the work and opening a PR, I thought I'd ask if there's any thoughts you would have?

**Comment by Majlanky** [comment]
Hello, thx for the issue. I remember that I was thinking about this some time ago. It should not be hard to solve it. I will look at it, and provide a solution 🍾 . Thx for the star also 😄

**Comment by Majlanky** [comment]
Fix will be provided in 2.4.1. Coming soon... Hope it will help you. If not, do not hesitate and open another issue ideally with steps to reproduce.
